ibm/mas_devops/roles/sls_cfg/templates/slscfg.yml.j2

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
None of the variables referenced in this template are configured inside this role.

Reply:
We are configuring  them in cfg-sls playbook, and we are validating them in task/main.yml.